examples/Cassie/cassie_utils.cc

```diff
@@ -197,11 +198,24 @@ const systems::SimCassieSensorAggregator& AddImuAndAggregator(
   const auto& accelerometer = Accelerometer<double>::AddToDiagram(
       body, X_BS, plant.gravity_field().gravity_vector(), plant, builder);
 
+  std::vector<int> joint_selections = {7,  8,  9,  10, 11, 12, 13,
+                                       14, 15, 16, 17, 18, 20, 22};
+
+  std::vector<int> ticks_per_revolution = {
```

Comment:
Also need to use drake indexing functions here so make sure the joints and the resolutions are in the same order